addons/binding/org.openhab.binding.miio/README.md

```diff
@@ -0,0 +1,54 @@
+# Xiaomi Robot Vacuum Binding
+
+This Binding is used to control Xiaomi Mi IO devices.
```

Comment:
This first paragraph will later be presented at http://docs.openhab.org/addons/bindings.html

Please add a longer description of the functionality, specialization and limitations. The paragraph should be between 80 and 160 characters.

Reply:
Please add a link to the first occurrence of "Xiaomi Mi IO"

Reply:
Consider mentioning the token limitation.